src/main/resources/specification/fdk-dataservice-harvester.yaml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.2
+info:
+  title: FDK dataservice harvester
+  description: Tilgjengeliggjør dataservice-beskrivelser fra en dataservice-katalog ihht DCAT-AP-NO v2.0
+  version: '1.0'
+  contact:
+    name: Digitaliseringsdirektoratet
+    url: 'https://fellesdatakatalog.digdir.no'
+    email: fellesdatakatalog@digdir.no
+  license:
+    name: License of API
+    url: 'http://data.norge.no/nlod/no/2.0'
+tags:
+  - name: dcat-ap-no-catalogs
+    description: DCAT-AP-NO catalogs
+  - name: dcat-ap-no-dataservices
+    description: DCAT-AP-NO dataservices
+servers:
+  - url: 'https://dataservice-catalog.fellesdatakatalog.digdir.no/'
+    description: Produksjonsmiljø
+  - url: 'https://dataservice-catalog.staging.fellesdatakatalog.digdir.no/'
+    description: Testmiljø
+  - url: 'https://dataservice-catalog.demo.fellesdatakatalog.digdir.no/'
+    description: Demomiljø
+paths:
+  /catalogs:
+    get:
+      tags:
+        - dcat-ap-no-catalogs
+      description: Returnerer samlinger av kataloger
+      operationId: getCatalogs
+      responses:
+        '200':
+          description: OK
+          content:
+            text/turtle:
+              schema:
+                type: string
+            text/n3:
+              schema:
+                type: string
+            application/rdf+json:
+              schema:
+                type: string
+            application/ld+json:
+              schema:
+                type: string
+            application/rdf+xml:
+              schema:
+                type: string
+            application/n-triples:
+              schema:
+                type: string
```

Comment:
Feilmeldinger?

Reply:
(gjelder alle endepunktene egentlig)

Reply:
1cae625

La til 404 på getById, ellers er det vel bare 500 og not-acceptable som er mulige feilmeldinger. Er det nødvendig å definere dei?

Reply:
Hm.. Hva tenker du @stigbd ? Sånn i utgangspunktet tenker jeg at alle mulige svar burde være dokumentert, men en 500 er vel å ta litt hardt i. not acceptable burde ha en beskrivelse (hva er det som er accepatble feks)

Reply:
Alt som er acceptable står beskrevet rett ovenfor...